libglusterfs/src/glusterfs/mem-pool.h

```diff
@@ -46,9 +46,8 @@ struct mem_acct_rec {
     const char *typestr;
     uint64_t size;
     uint64_t max_size;
-    uint64_t total_allocs;
-    uint32_t num_allocs;
     uint32_t max_num_allocs;
+    gf_atomic_t num_allocs;
     gf_lock_t lock;
```

Comment:
is there a use for this lock outside '#ifdef DEBUG' ?

Reply:
It is not required, i will update in next patch

Reply:
done